src/Event/Event.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Event;
+
+if (class_exists(\Symfony\Contracts\EventDispatcher\Event::class)) {
```

Comment:
please add some comment why it's needed and when it would not be needed anymore (how to cleanup, when to cleanup)

Reply:
Done :)